com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumePerSceneData.cs

```diff
@@ -19,50 +12,51 @@ enum ProbeVolumeState
 
     [ExecuteAlways]
     [AddComponentMenu("")] // Hide.
-    internal class ProbeVolumePerSceneData : MonoBehaviour, ISerializationCallbackReceiver
+    public class ProbeVolumePerSceneData : MonoBehaviour, ISerializationCallbackReceiver
```

Comment:
This needs to be public so RPs can strip it depending on settings. I've hidden everything else in the class instead, though.

Reply:
We then need a minimal doc to make it pass validation on top of the class

Reply:
Yup, in case you didn't know, you can easily check missing API doc by loading the HDRP_Test project and going Window=>Doc Checker